Gemfile.lock

```diff
@@ -34,43 +34,43 @@ GEM
       money (>= 6.0.0)
       railties (>= 3.2.0)
     erubi (1.9.0)
-    i18n (1.8.2)
+    i18n (1.8.3)
       concurrent-ruby (~> 1.0)
     json (2.3.0)
-    jwt (2.1.0)
-    jwt_signed_request (2.5.1)
-      jwt (>= 1.5.0, < 2.2.0)
+    jwt (2.2.1)
+    jwt_signed_request (2.5.4)
+      jwt (>= 1.5.0)
       rack
-    loofah (2.4.0)
+    loofah (2.6.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
     method_source (1.0.0)
     mini_portile2 (2.4.0)
-    minitest (5.14.0)
-    money (6.13.6)
+    minitest (5.14.1)
+    money (6.13.7)
```

Comment:
### [money](https://rubymoney.github.io/money)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 6.13.6 → 6.13.7

[_[change-log](https://github.com/RubyMoney/money/blob/master/CHANGELOG.md), [source-code](https://github.com/RubyMoney/money/)_]





<details>
<summary>Commits</summary>

A change of **5** commits. See the full changes on [the compare page](https://github.com/RubyMoney/money/compare/v6.13.6...v6.13.7).

These are the individual commits:
- (9eb5615) [Only warn once if defaults are not explicitly set (#902)](https://github.com/RubyMoney/money/commit/9eb5615924313745c6ec20b6c5927e5b16dfa53a)
- (5504993) [Fixes to the Money class docs (#903)](https://github.com/RubyMoney/money/commit/55049933cb8376ed7934c2829ba2a6c44c8997c7)
- (a3b1bf4) [Rounding Documentation Update (#904)](https://github.com/RubyMoney/money/commit/a3b1bf4a9a7c1b4dc8cd03f63e1c45f1b079f1ca)
- (b25f5a2) [Include old (current) values in the deprecation warnings (#9…](https://github.com/RubyMoney/money/commit/b25f5a232d8fadc43f7b9519df72cba3697be68a)
- (d00a84d) [Update version to 6.13.7](https://github.com/RubyMoney/money/commit/d00a84dee9da8b1e06654507d1e076286627cf03)


</details>